elastic/detection-rules
# [New Rule] New Microsoft 365 Impossible Travel Rules and Deprecation

# Pull Request

*Issue link(s)*:
* https://github.com/elastic/enhancements/issues/22525
* https://github.com/elastic/detection-rules/pull/4054



## Summary - What I changed
Added 2 new prebuilt OOTB detection rules for Microsoft 365 integration related to impossible travel activity. Also deprecated a [rule](https://github.com/elastic/detection-rules/blob/main/rules/integrations/o365/initial_access_microsoft_365_impossible_travel_activity.toml) as it is no longer applicable based on Defender changes to Microsoft 365 customers. Please see related issue for more details regarding this. 

Note: While it is straight-forward to identify impossible travel, identifying valid account compromise at login regardless of time are a bit more challenging, therefore the New Terms rule has been added to at least alert when any new country is logged from. This logic could be adjusted to be more region or city focused, but to reduce noise, country was selected.



## How To Test
<img width="2317" alt="Screenshot 2024-09-04 at 12 58 25 PM" src="https://github.com/user-attachments/assets/12ca21e1-19a2-41cc-a0e7-7073bedc30bb">

Query has been tested to match login behavior. Rules are threshold and new terms in order to identify logins from 2 separate locations within a 15 minute window of eachother and any time an unusual region appears as the source for a user login.



## Checklist




## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?
